static/app/views/detectors/components/forms/metric.tsx

```diff
@@ -0,0 +1,642 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import {Flex} from 'sentry/components/container/flex';
+import {Button} from 'sentry/components/core/button';
+import NumberField from 'sentry/components/forms/fields/numberField';
+import SegmentedRadioField from 'sentry/components/forms/fields/segmentedRadioField';
+import SelectField from 'sentry/components/forms/fields/selectField';
+import SentryMemberTeamSelectorField from 'sentry/components/forms/fields/sentryMemberTeamSelectorField';
+import Form from 'sentry/components/forms/form';
+import type FormModel from 'sentry/components/forms/model';
+import {SearchQueryBuilder} from 'sentry/components/searchQueryBuilder';
+import type {FilterKeySection} from 'sentry/components/searchQueryBuilder/types';
+import PriorityControl from 'sentry/components/workflowEngine/form/control/priorityControl';
+import {useFormField} from 'sentry/components/workflowEngine/form/hooks';
+import {Container} from 'sentry/components/workflowEngine/ui/container';
+import Section from 'sentry/components/workflowEngine/ui/section';
+import {IconAdd} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {TagCollection} from 'sentry/types/group';
+import {
+  ALLOWED_EXPLORE_VISUALIZE_AGGREGATES,
+  FieldKey,
+  FieldKind,
+  MobileVital,
+  WebVital,
+} from 'sentry/utils/fields';
+import {
+  AlertRuleSensitivity,
+  AlertRuleThresholdType,
+} from 'sentry/views/alerts/rules/metric/types';
+
+type MetricDetectorKind = 'threshold' | 'change' | 'dynamic';
+
+export function MetricDetectorForm({model}: {model: FormModel}) {
+  return (
+    <Form hideFooter model={model}>
+      <ChartContainer>
+        <MockChart />
+      </ChartContainer>
+      <FormStack>
+        <DetectSection />
+        <AssignSection />
+        <PrioritizeSection />
+        <ResolveSection />
+        <AutomateSection />
+      </FormStack>
+    </Form>
+  );
+}
+
+function ResolveSection() {
+  const kind = useFormField<MetricDetectorKind>('kind')!;
+
+  return (
+    <Container>
+      <Section
+        title={t('Resolve')}
+        description={
+          kind === 'dynamic'
+            ? t(
+                'Sentry will automatically resolve the issue when the trend goes back to baseline.'
+              )
+            : undefined
+        }
+      >
+        {kind !== 'dynamic' && (
+          <ThresholdField
+            flexibleControlStateSize
+            inline={false}
+            label={t('Close an incident when the value dips below:')}
+            placeholder="0"
+            name="resolve_threshold"
+            suffix="s"
+          />
+        )}
+      </Section>
+    </Container>
+  );
+}
+
+function AutomateSection() {
+  // const {openDrawer, isDrawerOpen} = useDrawer();
+  // const showDetails = () => {}
```

Comment:
Remove extra comments? We can add the automation drawer in a future PR

Reply:
Fixed in 93dbdb96c51 ✨